ci/pipeline.yml

```diff
@@ -125,35 +125,34 @@ jobs:
     - { trigger: false,                    get: bats, timeout: *timeouts-long }
     - { trigger: false,                    get: bosh-deployment, timeout: *timeouts-long }
 
-  - put: terraform-cpi
+  - task: terraform-apply
     tags: [ "openstack" ]
     timeout: *timeouts-long
-    params:
-      env_name: bats-ubuntu-manual
-      terraform_source: bosh-openstack-cpi-release/ci/terraform/ci/bats-manual
-      vars: &bats-terraform-vars
-        prefix: "bats-ubuntu-manual"
-        auth_url: ((concourse_openstack_auth.auth_url))
-        domain_name: ((concourse_openstack_auth.openstack_domain))
-        user_name: ((config-json.openstack_username))
-        password: ((config-json.openstack_password))
-        project_name: ((config-json.openstack_project))
-        cacert_file: ((config-json.tf_ca_file_path))
-        region_name: ((config-json.tf_region_name))
-        primary_net_name: "bats-ubuntu-manual-primary"
-        primary_net_cidr: "10.0.4.0/24"
-        primary_net_allocation_pool_start: "10.0.4.200"
-        primary_net_allocation_pool_end: "10.0.4.254"
-        secondary_net_name: "bats-ubuntu-manual-secondary"
-        secondary_net_cidr: "10.0.5.0/24"
-        secondary_net_allocation_pool_start: "10.0.5.200"
-        secondary_net_allocation_pool_end: "10.0.5.254"
-        ext_net_name: ((config-json.tf_external_network_name))
-        ext_net_id: ((config-json.tf_external_network_id))
-        ext_net_cidr: ((config-json.tf_ext_net_cidr))
-        dns_nameservers: ((config-json.tf_dns_nameservers))
-        concourse_external_network_cidr: null
-        openstack_default_key_public_key: ((config-json.tf_default_public_key))
+    image: terraform-image
```

Comment:
Need to "get" this in the job

Reply:
I'm now getting that image, and have tagged all other gets in that Job with "openstack".